pages/nft-book-store/collection/new.vue

```diff
@@ -186,15 +186,13 @@
       </UCard>
 
       <StripeConnectCard
-        :is-stripe-connect-checked="isStripeConnectChecked"
+        v-model="isStripeConnectChecked"
```

Comment:
name this as `v-model:isStripeConnectChecked` also?

Reply:
[🎨 Improve v-model bindings for clarity](https://github.com/likecoin/nft-book-press/pull/139/commits/dc3caa7654325d5af46e597b78898d21dc2e06e6)